apps/financial-aid/backend/src/app/modules/municipality/municipality.controller.ts

```diff
@@ -69,4 +69,27 @@ export class MunicipalityController {
       input,
     )
   }
+
+  @Put('id/:id')
```

Comment:
Should this endpoint be more explicitly named since it is doing a pretty explicit action

Reply:
Yess it is now, however if we want to add some other updates then it is easier if we have it like this, and it is consistent with other update functions, should I change it or? 🙃 🈴

Reply:
change to activity